options/ngeox.js

```diff
@@ -834,6 +834,22 @@ ngeox.QuerySource.prototype.wfsQuery;
 
 
 /**
+ * Options to configure the scale selector
+ * @typedef {{
+ *     dropup: (boolean|undefined)
+ * }}
+ */
+ngeox.ScaleselectorOptions;
+
+
+/**
+ * True to get a drop menu that opens imself to the top, instead of the bottom.
+ * @type {boolean|undefined}
+ */
+ngeox.ScaleselectorOptions.prototype.dropup;
```

Comment:
not sure that this line is needed (ping @gberaudo)

Reply:
That allow to have a documentation on each options, like https://camptocamp.github.io/ngeo/master/apidoc/ngeox.html#QueryableDataSources